### PR TITLE
[EDA] Program Auto Enrollment Recommended Fix Updates

### PR DESCRIPTION
**Pull request by skosgi1**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -1732,7 +1732,7 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Recommended fix when Health Check can&apos;t find a mapping&apos;s Auto-Enrollment Role</shortDescription>
-        <value>In EDA Settings, on the Affiliation Mappings subtab of the Affiliations tab, change the Auto-Enrollment Role to a valid, active picklist value for the Role field on the Affiliation object.</value>
+        <value>The auto-enrollment Role isn't valid. In EDA Settings | Program Enrollments, edit the Auto-Enrollment Program Mapping that specifies this Role to a valid, active picklist value for the Role field on the Affiliation object.</value>
```
Comment (xinnanlisf) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:1735]: Should we use &apos instead?

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 2]
```diff
@@ -1780,7 +1780,7 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Recommended fix when Health Check can&apos;t find a mapping&apos;s Auto-Enrollment Status</shortDescription>
-        <value>In EDA Settings, on the Affiliation Mappings subtab of the Affiliations tab, change the Auto-Enrollment Status to a valid, active picklist value for the Status field on the Affiliation object.</value>
```
Comment (xinnanlisf) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:1783]: This isn't matching the spec but i feel like there is a typo in the spec.

File: force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (skosgi1) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml]: Fixed it!

**Review by xinnanlisf — APPROVED** [review]

**Comment by spinkelman** [comment]
QA Passes - merging

**Pull request merged**